cmake/common/helpers_common.cmake

```diff
@@ -472,6 +472,9 @@ function(add_obs_plugin target)
     set(found_architecture TRUE)
   else()
     foreach(architecture IN LISTS _AOP_ARCHITECTURES)
+      if (${architecture} STREQUAL x86_64)
+      	set(architecture "amd64") # the x86_64 is called amd64 on BSDs
+      endif()
```

Comment:
This breaks Linux, because the `CMAKE_SYSTEM_PROCESSOR` is not `amd64` on those systems and this change unequivocally changes the value for every loop iteration.

Due to this, `obs-nvenc` and `obs-qsv11` are now excluded from Linux builds (as can be seen in the CI runs).

I think it's fine to just explicitly add `amd64` as a desired architecture to the `add_obs_plugin` calls.